extensions/amp-story/0.1/amp-story.js

```diff
@@ -215,8 +218,64 @@ export class AmpStory extends AMP.BaseElement {
 
     this.win.document.addEventListener('mozfullscreenchange',
         () => { this.onFullscreenChanged_(); });
+
+    this.desktopMedia_ = this.win.matchMedia('(min-width: 768px)');
+
+    this.boundOnResize_ = debounce(this.win, () => this.onResize(), 300);
+    this.getViewport().onResize(this.boundOnResize_);
+    this.onResize();
+
+    const doc = this.element.ownerDocument;
+    const n = doc.createElement('button');
+    n.classList.add('i-amphtml-story-button-move','i-amphtml-story-button-next');
+    this.element.appendChild(n);
+    const p = doc.createElement('button');
+    p.classList.add('i-amphtml-story-button-move','i-amphtml-story-button-prev');
+    this.element.appendChild(p);
+
+    const jsonLd = getJsonLd(doc);
+    if (jsonLd && jsonLd['@type'] === 'NewsArticle') {
+      const publisherInfo = jsonLd.publisher;
+      const logoInfo = publisherInfo.logo;
+      const logo = createElementWithAttributes(doc, 'amp-img', {
+        width: Math.floor(logoInfo.width / 2),
```

Comment:
Why are these divided by 2?

Reply:
I think the asset I was using from the ld+json was intended to be displayed on retina devices at half resolution, since it looked a little blurry at full width. Seems like a config/style issue though, and not something the code should handle. I'll update this.